src/widgets/refdata/list-ui.vue

```diff
@@ -2,35 +2,21 @@
 <template>
   <div row>
     <!-- ElasticSearch widget -->
-    <b-form-group :class="isRequired ? 'required' : ''" id="fundertype-form-group" horizontal :label-cols="uiLabel ? labelCols : 1" :description="uiDescription" :label="uiLabel">
-	  <b-input-group>
+    <b-form-group class="list-ui-form-group" :class="isRequired ? 'required' : ''" :label-cols="uiLabel ? labelCols : 1" :description="uiDescription" :label="uiLabel">
+      <b-input-group>
         <div v-if="type === 'multiselect'" class="flex-grow-1">
           <Multiselect v-model="model" @input="setValue" :options="items" v-if="items" :customLabel="customLabel"
             :optionsLimit="40" :allowEmpty="!isRequired" :showLabels="false" />
         </div>
         <b-input-group-append>
-          <b-btn variant="danger" ref="refErrorButton" id="refdata-error-btn" v-b-tooltip.hover="error" v-if="error">
+          <b-btn variant="danger" ref="refErrorButton" v-b-tooltip.hover="error" v-if="error">
```

Comment:
do not remove id field, as that enables testing. renaming is ok.

Reply:
I brought them back and renamed them but they won't be unique if there are multiple list-ui components. That would be a bit more work that I'd rather do in some other pr.